<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddContractWindow</class>
 <widget class="QDialog" name="AddContractWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>650</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>650</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Создание договора</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resource/resource.qrc">
    <normaloff>:/icons/icons/window-logo.ico</normaloff>:/icons/icons/window-logo.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
	background-color: white;
	border-top: 1px solid #9e9e9e;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="leftMargin">
      <number>40</number>
     </property>
     <property name="rightMargin">
      <number>40</number>
     </property>
     <item>
      <widget class="QGroupBox" name="contractNumberGroup">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QGroupBox {
	background: white;
	border-top: 1px solid black;
	border-bottom: 2px solid black;
	padding-right: 15px;
	padding-left: 15px;
	margin-top: 25px;
}</string>
       </property>
       <property name="title">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="topMargin">
         <number>9</number>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Договор на поставку товаров&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#0081c2;&quot;&gt;Договор №&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="contractNumber">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
	border: 1px solid black;
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="supplierGroup">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QGroupBox {
	border: 0px;
	background: white;
}

QLineEdit {
	background: #dae8fc;
	border: 1px solid #92abd0;
}

QLineEdit#conclusionDate {
	margin-bottom: 12px;
}

QLabel#conclusionDateLabel {
	padding-right: 5px;
	margin-bottom: 12px;
}

QLabel#expirationDateLabel {
	padding-right: 5px;
}
</string>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>20</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <property name="verticalSpacing">
         <number>0</number>
        </property>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Поставщик</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QDateEdit" name="completionDate">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>20</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="3" rowspan="2">
         <widget class="QDateEdit" name="conclusionDate">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>20</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="expirationDateLabel">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit {
background:white;
}


QLineEdit:focus{
background:#e9f3fb;
}</string>
          </property>
          <property name="text">
           <string>Дата окончания</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QComboBox" name="suppliers">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Поставщики, с которыми ещё не заключён действующий договор&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="conclusionDateLabel">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Дата заключения</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="description">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QScrollBar:vertical {
      border: 1px solid #77bdb0;
      background: white;
      width: 20px;
      margin: 35px 0 35px 0;
  }

  QScrollBar::handle:vertical {
      background: #e0dcdc;
	  border-top: 1px solid #77bdb0;
	  border-bottom: 1px solid #77bdb0; 
      min-height: 5px;
  }

  QScrollBar::handle:vertical:hover {
       background: #F19CBB;
  }

  QScrollBar::handle:vertical:pressed {
       background: #FDDB6D;
  }

  QScrollBar::add-line:vertical {
      border: 1px solid grey;
      background: white;
      height: 33px;
      subcontrol-position: bottom;
      subcontrol-origin: margin;
  }

  QScrollBar::add-line:vertical:hover {
      background: #CD9575;
  }

  QScrollBar::add-line:vertical:pressed {
      background: #FDDB6D;
  }

  QScrollBar::sub-line:vertical {
      border: 1px solid grey;
      background: white;
      height: 33px;
      subcontrol-position: top;
      subcontrol-origin: margin;
  }

  QScrollBar::sub-line:vertical:hover {
      background: #CD9575;
  }

  QScrollBar::sub-line:vertical:pressed {
      background: #FDDB6D;
  }

  QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical {
      border: 1px solid grey;
      width: 3px;
      height: 3px;
      background: white;
  }</string>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::AdjustIgnored</enum>
       </property>
       <property name="lineWrapMode">
        <enum>QPlainTextEdit::WidgetWidth</enum>
       </property>
       <property name="plainText">
        <string>1. ПРЕДМЕТ ДОГОВОРА, КОЛИЧЕСТВО, СРОКИ, ЦЕНА
1.1. Поставщик обязуется поставить Покупателю, а Покупатель принять и оплатить товары. 

2. ПОРЯДОК ПОСТАВКИ И ПРИЕМКИ ТОВАРА
2.1. Покупатель получает товар самовывозом со склада Поставщика.
2.2. Приемка продукции по качеству и количеству осуществляется в соответствии с товарно-транспортной накладной.
2.3. В случае выявления скрытых дефектов, Покупатель обязан в 3-дневный срок вызвать представителя Поставщика для оформления возврата.
2.4. В случае замены Поставщиком забракованного товара об этом составляется двусторонний акт и возврат оформляется накладной.

3. ПОРЯДОК РАСЧЕТОВ И ИМУЩЕСТВЕННАЯ ОТВЕТСТВЕННОСТЬ
3.1. Условия оплаты: Расчеты за товар, поставляемый по настоящему Договору, производятся путем перечисления денежных средств на р/с Поставщика по платежному поручению.
Датой платежа признается дата проводки денежных средств банком Поставщика.
3.2. В случае несвоевременной оплаты поставленного товара Покупатель уплачивает Поставщику пени в размере 0,4% от суммы договора за каждый день просрочки.</string>
       </property>
       <property name="backgroundVisible">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="printSaveGroup">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QGroupBox {
	background: white;
	border: 0px;
}

QPushButton {
	padding-left: 30px;
	padding-right: 30px;
	padding-top: 2px;
	padding-bottom: 2px;
	background-color: #f5f5f5;
	border-style: solid;
	border-width: 1px;
	border-color: #9e9e9e;
}

QPushButton#save {
	padding-left: 15px;
	padding-right: 15px;
}

 QPushButton:hover {
     background-color: #78DBE2;
 }
 QPushButton:pressed {
     background-color: #3F888F;
 }</string>
       </property>
       <property name="title">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>20</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="print">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Печать договора&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>Печать</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="save">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Сохранить</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resource/resource.qrc"/>
 </resources>
 <connections/>
</ui>
